config/locales/en.yml

```diff
@@ -832,7 +837,7 @@ en:
         publish/course_study_mode_form:
           attributes:
             study_mode:
-              blank: "Select full time or part time"
+              blank: Select a study pattern
```

Comment:
I saw on the card @RebeccaCryan  suggested "You must choose a study pattern. Select all that apply". Was there a change of mind?

Reply:
Yes, Slack thread: https://ukgovernmentdfe.slack.com/archives/C070ANGEZ8V/p1715964090673909